### PR TITLE
Advertise daisy's prow instance to our prow in the wild list

### PR DESCRIPTION
**Pull request by krzyzacy**
(looks like their deck is not working though, but other stuff is running :-p )

/assign @cjwagner

### CODE REVIEW

File: prow/README.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ Prow is used by the following organizations and projects:
 - [Azure acs-engine](https://github.com/Azure/acs-engine/tree/master/.prowci)
 - [tensorflow/minigo](https://github.com/tensorflow/minigo#automated-tests)
 - [helm/charts](https://github.com/helm/charts)
+- [daisy(google compute image tools)](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/test-infra#prow-and-gubenator)
```
Comment (cjwagner) [inline prow/README.md]: super-nit: I think Daisy is typically capitalized.

**Comment by stevekuznetsov** [comment]
@midnightconman @darwin67 @alvaroaleman if you want to advertise yours, this would be a good place :)

File: prow/README.md (hunk 1 above)
Comment (krzyzacy) [reply prow/README.md]: :+1:

**Comment by alvaroaleman** [comment]
> @midnightconman @darwin67 @alvaroaleman if you want to advertise yours, this would be a good place :)

We don't have anything to show currently as we don't have a public Deck instance because its not possible to hide periodics yet. Once that is the case I'll certainly add us there :)

**Comment by darwin67** [comment]
> > @midnightconman @darwin67 @alvaroaleman if you want to advertise yours, this would be a good place :)
> 
> We don't have anything to show currently as we don't have a public Deck instance because its not possible to hide periodics yet. Once that is the case I'll certainly add us there :)

same here! :)

**Pull request merged**